crates/bevy_dev_tools/src/ci_testing/mod.rs

```diff
@@ -45,9 +46,11 @@ impl Plugin for CiTestingPlugin {
                 fixed_frame_time,
             )));
         }
-
         app.add_event::<CiTestingCustomEvent>()
             .insert_resource(config)
-            .add_systems(Update, systems::send_events);
+            .add_systems(
+                Update,
+                systems::send_events.before(bevy_window::close_when_requested),
```

Comment:
This is kinda outside the scope of this PR, and now useless as there has been a new conflict on Update...